src/features/dsPreview/state/dsPreviewActions.ts

```diff
@@ -3,10 +3,15 @@ import { ApiAction, ApiActionThunk, CALL_API } from "../../../store/api";
 
 export function loadDsPreview(ref: QriRef): ApiActionThunk {
   return async (dispatch, getState) => {
-    return dispatch(fetchDsPreview(ref))
+    await Promise.all([dispatch(fetchDsPreview(ref))])
+    dispatch(fetchDsPreviewBody(ref))
+    dispatch(fetchDsPreviewReadme(ref))
+    return dispatch(fetchPreviewDone(ref))
   }
 }
 
+export const bodyPageSizeDefault = 50
```

Comment:
copy from `datasetActions`, we should think about just importing those here and reusing them.